Tests/Marketplace/marketplace_services.py

```diff
@@ -389,14 +395,41 @@ def all_levels_dependencies(self):
         return self._all_levels_dependencies
 
     @property
-    def statistics_metadata(self):
-        return {
-            Metadata.DOWNLOADS: self.downloads_count,
-            Metadata.SEARCH_RANK: self._search_rank,
+    def update_metadata(self):
+        """
+        Returns a dictionary containing updated metadata fields.
+        This function updates the statistics_metadata fields (downloads, searchRank, tags, and integrations).
+        If is_metadata_updated is True, it also updates the fields that are not listed in PACK_METADATA_REQUIRE_RN_FIELDS.
+        Returns:
+            dict: Updated metadata fields.
+        """
+        update_statistics_metadata = {
+            Metadata.CREATED: self._create_date,
+            Metadata.UPDATED: self._update_date,
```

Comment:
Don't need these fields to be updated for is_metadata_updated packs.
Please remove.